openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -358,3 +364,22 @@ def get_all_course_keys(cls):
             CourseKey.from_string(course_overview['id'])
             for course_overview in CourseOverview.objects.values('id')
         ]
+
+    def is_discussion_tab_enabled(self):
+        """
+        Returns True if course has discussion tab and is enabled
+        """
+        course_overview_tabs = self.course_overview_tab.all()
+        # creates circular import; hence explicitly imported is_discussion_enabled
+        for tab in course_overview_tabs:
+            if tab.tab_id == "discussion" and django_comment_client.utils.is_discussion_enabled(self.id):
+                return True
+        return False
+
+
+class CourseOverviewTab(models.Model):
+    """
+    Model for storing and caching tabs information of a course.
+    """
+    tab_id = models.CharField(max_length=50)
+    course_overview = models.ForeignKey(CourseOverview, db_index=True, related_name="course_overview_tab")
```

Comment:
The `related_name` should be short like `tabs`.  No need to repeat `course_overview` in the related_name the tabs would be access through a course_overview.